packages/babel-jest/src/index.js

```diff
@@ -135,16 +135,7 @@ const createTransformer = (options: any): Transformer => {
       // babel v7 might return null in the case when the file has been ignored.
       const transformResult = babelTransform(src, theseOptions);
 
-      if (!transformResult) {
-        return src;
-      }
-
-      const shouldReturnCodeOnly =
-        transformOptions == null ||
-        transformOptions.returnSourceString == null ||
-        transformOptions.returnSourceString === true;
-
-      return shouldReturnCodeOnly ? transformResult.code : transformResult;
+      return transformResult || src;
```

Comment:
To keep the output of the `babel-jest` consistent, we might want to return an object when the babelTransform returned undefined.

```javascript
//instead of this
return transformResult || src;
//We may have something like
return transformResult || {code: src, map: null};
```

Although I dunno under what circumstances this wouldn't return anything, given that we make sure it won't.
https://github.com/SimenB/jest/blob/d5427430e281a1230cd2b9a27aa3ea6d51fa7332/packages/babel-jest/src/index.js#L115